src/hibernate-resume/hibernate-resume.c

```diff
@@ -1,58 +1,76 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
 #include <errno.h>
-#include <stdio.h>
 #include <sys/stat.h>
 
-#include "alloc-util.h"
 #include "devnum-util.h"
 #include "fileio.h"
 #include "initrd-util.h"
 #include "log.h"
+#include "main-func.h"
+#include "parse-util.h"
+#include "static-destruct.h"
+#include "string-util.h"
 
-int main(int argc, char *argv[]) {
+static char *arg_resume_device = NULL;
```

Comment:
if this points into argv[] you might as well make it `const char*`, given this should not be changed (at least not via this pointer I think)